packages/compiler-cli/src/ngtsc/util/src/local_compilation.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {NgCompilerOptions} from '../../core/api';
+
+/** Cannonical logic as to whether the compilation is in local mode. */
```

Comment:
typo for canonical, and I'd propose something like:

```suggestion
/** Gets whether local compilation is enabled in the compiler options. */
```